src/Tabs.js

```diff
@@ -69,6 +74,7 @@ class Tabs extends Component {
                 </li>
               );
             })}
+            <div className={indicatorClasses} style={{ zIndex: 1 }} />
```

Comment:
Hey @brunogfranca, this should be an `li` I think

Reply:
We could also use here the `cx` method:
`<li className={cx('indicator', indicatorClasses)} style={{ zIndex: 1 }} />`

Reply:
I made it as a div (even though it's not proper html) because it was the popular fix to this issue. Haven't even consider testing if the li would work (just assumed it didn't). Tested it now and works just fine. I'll change it here 👍